patches/chromium/skid-proof_video_decoding_memory_use.patch

```diff
@@ -17,10 +17,10 @@ re-evaluate current state of chromium once we upgrade and drop this fix
 if they put something better in.
 
 diff --git a/media/base/BUILD.gn b/media/base/BUILD.gn
-index 80cc2864762202bcf4211e8682d0fec3057bc655..2182186c0dca1be58caf4bba237d7ae8cddbd622 100644
+index 54e6842abd4e4041f53e75d0bcc4e0ee72d3d453..8b66eb082431ac57ea779a10ece6fc9ee46b7d9a 100644
 --- a/media/base/BUILD.gn
 +++ b/media/base/BUILD.gn
-@@ -313,6 +313,8 @@ jumbo_source_set("base") {
+@@ -329,6 +329,8 @@ source_set("base") {
```

Comment:
idk how this diff got into the patch :thinking: will check in the morning

Reply:
That sounds plausible - Chromium got rid of jumbo builds a few years ago: https://groups.google.com/a/chromium.org/g/chromium-dev/c/DP9TQszzQLI